f5/bigip/tm/ltm/policy.py

```diff
@@ -56,6 +59,24 @@ def __init__(self, policy_s):
         self._meta_data['required_creation_parameters'].update(('strategy',))
         temp = {'tm:ltm:policy:rules:rulescollectionstate': Rules_s}
         self._meta_data['attribute_registry'] = temp
+        self._meta_data['optional_parameters'] = {'rules': ['description']}
+
+    def _filter_version_specific_options(self, tmos_ver, **kwargs):
```

Comment:
i think the point was missed here, i was talking about streamlining this into resource.py, but fine since this is a bigger job, just open an issue for it and assign it to me, i will do this later with update() rework